test/functional/update/slow-server/test.bats

```diff
@@ -8,6 +8,11 @@ port=""
 targetfile="6c27df6efcd6fc401ff1bc67c970b83eef115f6473db4fb9d57e5de317eba96e"
 
 setup() {
+  # Skip if not in travis env
+  if [ -z "${TRAVIS}" ]; then
+    skip "Skiping test because it's too slow"
```

Comment:
misspelled "skipping" and please add more details in the skip message (maybe "Skipping slow test for local development, test only runs in CI")